modules/nodes-pods-using-example.adoc

```diff
@@ -55,11 +55,11 @@ spec:
       volumeMounts: <5>
         - name: default-token-wbqsl
           readOnly: true
-          mountPath: /var/run/secrets/kubernetes.io/serviceaccount
+          mountPath: /var/run/secrets/kubernetes.io/serviceaccount <6>
       terminationMessagePolicy: File
-      image: registry.redhat.io/openshift4/ose-ogging-eventrouter:v4.3 <6>
-  serviceAccount: default <7>
-  volumes: <8> 
+      image: registry.redhat.io/openshift4/ose-ogging-eventrouter:v4.3 <7>
+  serviceAccount: default     <8>
+  volumes:                    <9>
```

Comment:
More of a question, but I see you're aligning these two callouts, but not the others. Is this intended?